Firestore/core/test/firebase/firestore/remote/serializer_test.cc

```diff
@@ -1573,7 +1643,39 @@ TEST_F(SerializerTest, DecodesListenResponseWithExistenceFilter) {
   proto.mutable_filter()->set_target_id(100);
 
   SCOPED_TRACE("DecodesListenResponseWithExistenceFilter");
-  ExpectWatchChangeDeserializationRoundTrip(model, proto);
+  ExpectDeserializationRoundTrip(model, proto);
+}
+
+TEST_F(SerializerTest, DecodesVersion) {
+  SnapshotVersion model(Version(123456789).timestamp());
```

Comment:
I'd create a local variable for the version.

Reply:
Done.